package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
This line removes the work within a recent PR https://github.com/bahmutov/rollem/pull/17.

See comments on peer dependency

A pull -r with a rebase atop master should do the trick. (or just replace the line :-) )